sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/Customized/Models/ApiCreateOrUpdateContent.Serialization.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ClientModel.Primitives;
+using System.Runtime.CompilerServices;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.ResourceManager.ApiManagement;
+
+[CodeGenSerialization(nameof(TermsOfServiceUri), DeserializationValueHook = nameof(DeserializetermsOfServiceUrlValue))]
+public partial class ApiData : IUtf8JsonSerializable, IJsonModel<ApiData>
+{
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    internal static void DeserializetermsOfServiceUrlValue(JsonProperty property0, ref Uri termsOfServiceUrl)
```

Comment:
let us rename this parameter from `property0` to `property`

Reply:
And there is a typo in the method name
```suggestion
    internal static void DeserializeTermsOfServiceUrlValue(JsonProperty property0, ref Uri termsOfServiceUrl)
```